yast/src/modules/SMTData.rb

```diff
@@ -0,0 +1,1480 @@
+          Builtins.sformat(
+            _(
+              "Server certificate %1 does not exist.\n" +
+                "Would you like to run CA management to create one?\n" +
+                "\n" +
+                "The server certificate is vitally important for the NU server to support SSL.\n"
+            ),
+            @server_cert
+          ),
+          _("&Run CA management"),
+          _("&Skip"),
+          @ca_already_called ? :focus_no : :focus_yes
+        )
+        # Package needs to be installed
+        if PackageSystem.CheckAndInstallPackagesInteractive([ca_mgm_package])
+          Builtins.y2milestone("Running ca_mgm")
+          @ca_already_called = true
+
+          Wizard.OpenNextBackDialog
+          progress_orig = Progress.set(false)
+
+          # bnc #400782
+          # Custom control file can make ProductControl not to find anything
+          previous_custom_control_file = ProductControl.custom_control_file
+          if previous_custom_control_file != ""
+            Builtins.y2milestone(
+              "Previous custom_control_file: %1",
+              previous_custom_control_file
+            )
+          end
+
+          # bnc #471162
+          # Use own control file (do not depend on the system one which might be missing)
+          ProductControl.custom_control_file = @smt_control_file
+          ProductControl.Init
+
+          # Call installation proposal (service: CA)
+          ret = WFM.call("inst_proposal", [{ "proposal" => "smt_ca" }])
+
+          Builtins.y2milestone(
+            "Restoring previous custom_control_file: %1",
+            previous_custom_control_file
+          )
+          ProductControl.custom_control_file = previous_custom_control_file
+          ProductControl.Init
+
+          Builtins.y2milestone("Service proposal returned: %1", ret)
+          Progress.set(progress_orig)
+          Wizard.CloseDialog
+
+          return false 
+          # Package is not installed and couldn't be installed
+        else
+          Report.Error(
+            Builtins.sformat(
+              _("Cannot run CA management because package %1 is not installed."),
+              ca_mgm_package
+            )
+          )
+          return true
+        end
+      else
+        Builtins.y2warning("User doesn't want to run CA management.")
+        return true
+      end
+    end
+
+
+    # Function migrates SMT between NCC and SCC
+    #
+    # @return [Boolean] if successful
+    def MigrateCustomerCenter
+      ret = {}
+      if Ops.get(@all_credentials, ["NU", "ApiType"], "NCC") == "NCC"
+        ret = Convert.convert(
+          SCR.Execute(
+            path(".target.bash_output"),
+            "/usr/sbin/smt-scc-ncc-migration"
+          ),
+          :from => "any",
+          :to   => "map <string, any>"
+        )
+      else
+        ret = Convert.convert(
+          SCR.Execute(
+            path(".target.bash_output"),
+            "/usr/sbin/smt-ncc-scc-migration"
```

Comment:
This command is also gone. The `SMT::NCCRegTools` module was removed in commit 5a62d8ceb a year ago. All these script requires that module and does not even compile without it.